src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -103,16 +103,27 @@
                      {:id "library:dataUseRestriction" :header (:title (:library:dataUseRestriction attributes))
                       :column-data :library:dataUseRestriction :initial-width 180}
                      {:id "library:numSubjects" :header (:title (:library:numSubjects attributes))
-                      :column-data :library:numSubjects :initial-width 100}]
+                      :column-data :library:numSubjects :initial-width 100}
+                     {:id "library:consentCodes" :header (:title (:library:consentCodes attributes))
+                      :column-data :library:consentCodes :initial-width 100
+                      :render (fn [data]
+                                (->> data
+                                     sort
+                                     (map #(style/render-tag {:style {:margin "0 0.1rem" :padding "0 0.5rem" :display "inline-flex"}} %))))}]
```

Comment:
I think the `:display "inline-flex"` is unnecessary. Also, since we're doing this in two places, let's make
```clojure
(def render-tag (partial style/render-tag {:style {:margin "0 0.1rem" :padding "0 0.5rem"}}))
```
so we can just `(map render-tag)` as the last item in this thread.

Reply:
I'm extracting this out to a private method since I have to use it in 3 places now. Will refactor there.

Reply:
The inline-flex is necessary to fix a weird CSS glitch. Without it, sometimes elements in the rendered tag list are invisible.